src/EnergyPlus/DXCoils.cc

```diff
@@ -7463,103 +7463,75 @@ namespace DXCoils {
                 if (DXCoil(DXCoilNum).MSRatedSHR(Mode) == AutoSize) {
                     IsAutoSize = true;
                 }
-                if ( Mode == DXCoil( DXCoilNum ).NumOfSpeeds ) {
-                    if ( CurSysNum > 0 ) {
```

Comment:
So this CurSysNum check is no longer needed?

Reply:
No, RequestSizing handles all of that.

Reply:
Excellent.  I'm building and running some tests locally and we'll try to get this merged ASAP.